src/script/page/RightSidebar/AddParticipants/AddParticipants.tsx

```diff
@@ -200,7 +205,7 @@ const AddParticipants: FC<AddParticipantsProps> = ({
           placeholder={t('addParticipantsSearchPlaceholder')}
         />
 
-        {showIntegrations && (
+        {showIntegrations && !isMLSEnabled && (
```

Comment:
Is that the correct state to check? I'm not sure but I expect this to return true if the conversation is Proteus but MLS is allowed on the team level

Reply:
Nice catch, @V-Gira.
Yes, you are right. Let me fix it.